search/tavily/main.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+import sys
+
+from tavily import TavilyClient
+from urllib3.util import parse_url
+
+
+def main():
+    if len(sys.argv) < 2:
+        print("Usage: python main.py [search | extract]")
+        sys.exit(1)
+
+    command = sys.argv[1]
+    client = TavilyClient()  # env TAVILY_API_KEY required
+
+    match command:
+        case "search":
+            query = os.getenv("QUERY").strip()
```

Comment:
better to read env safely here
`query = os.getenv("QUERY", "").strip()`

Reply:
done